goosebit/ui/bff/devices/routes.py

```diff
@@ -79,6 +79,8 @@ async def devices_patch(_: Request, config: DevicesPatchRequest) -> StatusRespon
             await updater.update_feed(config.feed)
         if config.force_update is not None:
             await updater.update_force_update(config.force_update)
+        if config.auth_token is not None:
```

Comment:
Again: dead code?

Reply:
Maybe?  Should we let the user set the token via UI in the future?  If so this should be here, makes 1 less thing we will have to search for in the future, and the performance hit is negligible.

Reply:
Used in tests.